backend/migrations/000021_add_achievements_details_column.down.sql

```diff
@@ -1,3 +0,0 @@
-ALTER TABLE user_achievements RENAME TO community_users_achievements;
-ALTER TABLE community_users_achievements DROP COLUMN details;
-
```

Comment:
duplicate of https://github.com/DapperCollectives/CAST/blob/375d108e3ef2b3cd0d11b2d8772ad295e8e6f6bf/backend/migrations/000020_add_achievements_details_column.down.sql#L0-L1